R/github-actions.R

```diff
@@ -17,25 +17,45 @@
 #' * `test-coverage`: Compute test coverage and report to
 #'    <https://about.codecov.io> by calling [covr::codecov()].
 #' * `pkgdown`: Automatically build and publish a pkgdown website.
-#'    But we recommend instead calling [use_pkgdown_github_pages()] which
-#'    performs other important set up.
-#' * `pr-commands`: Enables the use of two R-specific commands in pull request
-#'    issue comments: `/document` to run `roxygen2::roxygenise()` and
-#'    `/style` to run `styler::style_pkg()`. Both will update the PR with any
-#'    changes once they're done.
+#'    But we recommend instead calling [use_pkgdown_github_pages()], which
+#'    sets up the `pkgdown` workflow AND performs other important set up.
 #'
-#' If you call `use_github_action()` without arguments, you'll be prompted to
-#' pick from one of these. Otherwise you can see a complete list of
-#' possibilities provided by r-lib at
-#' <https://github.com/r-lib/actions/tree/v2/examples>, or you can supply
-#' your own `url` to use any other workflow.
+#' If you call `use_github_action()` without arguments, you'll get a choice of
+#' some recommended workflows. Otherwise you can specify the name of any
+#' workflow provided by `r-lib/actions`, which are listed at
+#' <https://github.com/r-lib/actions/tree/v2/examples>. Finally you can supply
+#' the full `url` to any workflow on GitHub.
 #'
-#' @param name For `use_github_action()`: Name of one of the example workflow
-#'   from <https://github.com/r-lib/actions/tree/v2/examples> (with or
-#'   without extension), e.g. `"pkgdown"`, `"check-standard.yaml"`.
+#' ## Other workflows
+#' Other specific workflows are worth mentioning:
+#' * `pr-commands`: `r lifecycle::badge("superseded")` Enables the use of two
+#'    R-specific commands in pull request issue comments: `/document` to run
+#'    `roxygen2::roxygenise()` and `/style` to run `styler::style_pkg()`. Both
+#'    will update the PR with any changes once they're done.
+#'
+#'    We don't recommend new adoption of the `pr-commands` workflow. Better
+#'    alternatives exist for code formatting (see below) and we plan to
+#'    re-implement documentation updates using a similar approach.
+#' * `format-suggest` and `format-check` from
```

Comment:
```suggestion
#' * `format-suggest` or `format-check` from
```

I view them more as an either or situation, depending on your preferences